packages/core/src/lib/icon/icons.ts

```diff
@@ -76,3 +79,10 @@ export const paths = {
  * The possible icon names that can be rendered. This is good for typing props.
  */
 export type IconName = keyof typeof paths;
+
+/** 
+ * A map of icon names, useful for dynamically adding icons with type safety.
+ */
+export const iconNames = Object.fromEntries<IconName>(
+  Array<IconName>.from(Object.keys(paths)).map((name) => [name, name as IconName])
+) as Record<IconName, IconName>
```

Comment:
Open to suggestions on a better way of easily using icon names dynamically, this came up in the radio and I know we do `icon` selection elsewhere. Usually that looks something like:

```svelte
let icon: IconName:
$: {
  if (thing) {
    icon = 'some-magic-string';
  } elseif {
    icon = 'another-magic-string';
  } else {
    icon = 'yet-another-magic-string';
  }
}
```

Don't love having all the magic strings, and while that works for singular icons it gets messier with determining icons for multiple elements like in the radio. This at least gives us a quick map of names we can refer to, and should make refactoring icon names slightly less painful in the future. If we like this (or something else that accomplishes the same thing), I can refactor other places where we declare icon names to use it as well for consistency.